README.md

```diff
@@ -28,7 +28,7 @@ Learn more at <http://hibernate.org/reactive>.
 
 Hibernate Reactive has been tested with:
 
-- Java 8
+- Java 11
```

Comment:
since the wording is "has been tested with" , shouldn't we list Java 16 and 17 as well?

Reply:
@DavideD up

Reply:
Added 11,16 and 17